src/plugins/console/public/lib/mappings/mappings.ts

```diff
@@ -137,17 +180,17 @@ export function getTypes(indices) {
     // multi index mode.
     Object.keys(perIndexTypes).forEach((index) => {
       if (!indices || indices.includes(index)) {
-        ret.push(getTypes(index));
+        ret.push(...getTypes(index));
```

Comment:
This change was made for better typing, it doesn't change the behavior of the function because the array is flattened before return, so the returned array remains same.